media/css/firefox/new/break-free.scss

```diff
@@ -3,7 +3,6 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 @import '../../pebbles/includes/lib';
-@import '../../pebbles/components/buttons-download';
```

Comment:
`buttons-download` is now included in global.scss, so they should no longer be needed here.